scripts/functions-deploy-tests/cli.ts

```diff
@@ -0,0 +1,56 @@
+import * as spawn from "cross-spawn";
+import { ChildProcess } from "child_process";
+
+interface Result {
+  proc: ChildProcess;
+  stdout: string;
+  stderr: string;
+}
+
+/**
+ * Execute a Firebase CLI command.
```

Comment:
Take a look at `scripts/integration-helpers/cli.ts`. I think you'll find it does what you want (or close)

Reply:
It was close, but I needed a little more control over stdout/stderr then what the other clit.s provided.

I think I could've spent few hours modifying the shared lib and add more flags to it, but out of laziness decided against it :(.

If you really hate what I did here, let me know.